src/frontend/js/utils/test/factories.ts

```diff
@@ -177,7 +177,7 @@ export const TargetCourseFactory = createSpec({
   code: faker.unique(faker.random.alphaNumeric(5)),
   organizations: [],
   title: faker.random.words(1, 3),
-  course_runs: derived(() => JoanieCourseRunFactory().generate(1, 3)),
+  course_runs: derived(() => JoanieCourseRunFactory().generate(3, 5)),
```

Comment:
To have more `course_runs` in every stories, it was made to avoid having to duplicate the generation of 3,5 course runs accros multiple stories. Hence it seemed to work well with the actual codebase, so why not? But I could revert this change and do it in stories if you want?